remo/profiles/forms.py

```diff
@@ -143,8 +143,9 @@ def clean(self):
 
         # Do not raise a validation error if the user belongs to the Alumni
         # group or has admin privileges
-        if (not mentor and not
-                self.request.user.groups.filter(name='Admin').exists()):
+        if (not mentor and
```

Comment:
this probably should be `and not`.

Reply:
Thinking about this, we probably can remove that whole block. As anybody else won't be able to change it, and for Admin and Council it indeed should be optional, we don't need it. Do you agree?